core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2853,7 +2853,7 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_PERIODIC_BLOCK_INTEGRITY_CHECK_REPAIR =
       booleanBuilder(Name.MASTER_PERIODIC_BLOCK_INTEGRITY_CHECK_REPAIR)
-          .setDefaultValue(false)
+          .setDefaultValue(true)
           .setDescription("Whether the system should delete orphaned blocks found during the "
               + "periodic integrity check. This is an experimental feature.")
```

Comment:
is this still experimental then if we set this to be on by default?

Reply:
@yuzhu believes that if we're going to pay the price of checking block integrity, we might as well repair it automatically if we find orphaned blocks. How experimental do we think this feature is? I'm not super familiar with it.

Reply:
my simple rec is to remove the experimental wording if we intend on running this by default. but double checking that it is indeed stable to turn on...